renku/core/management/git.py

```diff
@@ -302,12 +306,15 @@ def commit(
                 raise errors.NothingToCommit()
             return
 
-        argv = [os.path.basename(sys.argv[0])
-                ] + [remove_credentials(arg) for arg in sys.argv[1:]]
+        if commit_message is None or not isinstance(commit_message, str):
```

Comment:
It's good to check for the empty message as well. Also, for the case of non-string message, I'd rather raise an exception or just use `str(commit_message)` to make it visible instead of silently ignoring it. Since we are the user of this API, `str(commit_message)` would be enough.